webnode/config/webpack.config.prod.js

```diff
@@ -264,16 +184,7 @@ module.exports = {
         "eth-address": "0xD1833A50f411432aD38E8374df8Cfff79e743788"
       }
     }),
-    new webpack.EnvironmentPlugin({
-      NODE_ENV: "production",
-      DEBUG: false
-    }),
```

Comment:
keep this, used in the redux index.js file

Reply:
okay, ill add it back in